config/crds/hive_v1alpha1_clusterstate.yaml

```diff
@@ -0,0 +1,85 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  creationTimestamp: null
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: clusterstates.hive.openshift.io
+spec:
+  group: hive.openshift.io
+  names:
+    kind: ClusterState
+    plural: clusterstates
```

Comment:
IMO we should add a `shortNames` here as it will be used more often I guess. May be `cs`. But this is just a wish list and not blocker to merge the PR